about.md

```diff
@@ -236,7 +236,7 @@ Our members come from different schools in southern California.
 * Beacon Park School (K8)
 * Many More!
 
-If you are a 6th grade and above CS enthusiast who want to learn together with our team members,
+If you are a 6th grade and above CS enthusiast who wants to learn together with our team members,
 or if you are good at some areas that you want to share with our community by providing online courses.
 Come and join ICC by filling out [this form](https://forms.gle/Ky4S6YLkkursGyZn6).   
-Send an email to irvinecodingclub@gmail.com if you didn't get back from us.
+Send an email to <a href="mailto:irvinecodingclub@gmail.com">irvinecodingclub@gmail.com</a> if you didn't get back from us.
```

Comment:
Great find with the unlinked email! Are we trying to use admin@irvinecoding.club or irvinecodingclub@gmail.com tho?